test/e2e/test/elasticsearch/checks_data.go

```diff
@@ -132,3 +124,33 @@ func (dc *DataIntegrityCheck) Verify() error {
 	}
 	return nil
 }
+
+// dataIntegrityReplicas returns the number of replicas to use for the data integrity check,
+// according to the cluster topology, since it affects the cluster health during the mutation.
+func dataIntegrityReplicas(b Builder) int {
```

Comment:
Naming nit: given that this is a private function in a file with the sole purpose of setting up a data integrity check the `dataIntegrity` prefix seems somewhat redundant.

Reply:
The reason is the function is private to the package, not to the file. The package itself contains many other unrelated things, having a function called `replicas()` available in those files seems a bit misleading.